dev/build.example.testcontainers_fat/build.gradle

```diff
@@ -15,9 +15,22 @@
  // This bundle is required both at compiletime and runtime.
  addRequiredLibraries.dependsOn copyTestContainers
  
+ //TODO temporary while we switch from ojdbc8_g to ojdbc8
+ task copyJdbcDriversOracleRename(type: Copy) {
+  mustRunAfter jar
+  from configurations.jdbcDrivers
+  into new File(autoFvtDir, 'publish/shared/resources/jdbc')
+  rename 'jcc.*.jar', 'jcc.jar'
+  rename 'derby-.*.jar', 'derby.jar'
+  rename 'derbyclient-.*.jar', 'derbyclient.jar'
+  rename 'ojdbc8_g.*.jar', 'ojdbc8.jar'
```

Comment:
Why is copyJdbcDriversOracleRename only needed for ojdbc8 and not ojdbc11 as well?

Reply:
This was added just to test the temporary code added to fattest.simplicity.